models/product/src/product-variant/builders.spec.ts

```diff
@@ -63,7 +63,12 @@ const validateGraphqlModel = (
         }),
       ]),
       sku: expect.any(String),
-      attributesRaw: [],
```

Comment:
We now make sure we have at least one item in this list (as updated in the corresponding `fields-config` file).